ait/core/server/plugin.py

```diff
@@ -0,0 +1,138 @@
+from collections import defaultdict
+import importlib
+import re
+
+import gevent
+import gevent.monkey; gevent.monkey.patch_all()
+
+import ait.core
+from ait.core import  db, limits, log, notify, tlm
+from client import ZMQInputClient
+
+
+class Plugin(ZMQInputClient):
```

Comment:
I think this should also be an abstract base class. See `Handler` for additional notes.